2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc

```diff
@@ -382,7 +437,7 @@ Although the directory name is `roles`, and the files are called roles, these fi
 Check the contents of the OSPP profile Playbook in your editor and verify that a task for rule `accounts_tmout` is there among all the other tasks.
 
 ----
-$ gedit build/roles/ssg-rhel8-role-ospp.yml
+[... lab4_ansible]$ nano build/roles/ssg-rhel8-role-ospp.yml
 ----
 
 Keep in mind that even if the comments in the headers say that it is an Ansible Role, it is not an Ansible Role, it is an Ansible Playbook.
```

Comment:
The exercise somehow vanishes into nothingness - could you think of some sweet ending?

Reply:
I have tried to put something small at the end.